data-shaping/converter/data_model_mapping.py

```diff
@@ -0,0 +1,116 @@
+        "title": {"sheet": "tlkpEthnic", "column": "ECDESC"},
+    },
+    "DisabillityCategory": {
+        "id": {"sheet": "tlkpDisabilityCategory", "column": "DCID"},
+        "title": {"sheet": "tlkpDisabilityCategory", "column": "DCDESC"},
+    },
+    "Religion": {
+        "id": {"sheet": "tlkpReligion", "column": "RLID"},
+        "title": {"sheet": "tlkpReligion", "column": "RLREL"},
+    },
+    "SexualOrientation": {
+        "id": {"sheet": "tlkpSexor", "column": "SOID"},
+        "title": {"sheet": "tlkpSexor", "column": "SODESC"},
+    },
+    "Nationality": {
+        "id": {"sheet": "tlkpCountries", "column": "CTID"},
+        "title": {"sheet": "tlkpCountries", "column": "CTCNTR"},
+    },
+    "MaritalStatus": {
+        "id": {"sheet": "tlkpMaritalStatus", "column": "MRID"},
+        "title": {"sheet": "tlkpMaritalStatus", "column": "MRDESC"},
+    },
+    "FundingProvider": {
+        "id": {"sheet": "tlkpLETB", "column": "REID"},
+        "title": {"sheet": "tlkpLETB", "column": "RELETB"},
+    },
+    "HEIAward": {
+        "id": {"sheet": "tlkpHEI", "column": "HEID"},
+        "title": {"sheet": "tlkpHEI", "column": "HENM"},
+    },
+    "HSSTPathway": {
+        "id": {"sheet": "tlkpHSSTPathway", "column": "PTID"},
+        "title": {"sheet": "tlkpHSSTPathway", "column": "PTDESC"},
+    },
+    "Specialism": {
+        "id": {"sheet": "tlkpSpecialism", "column": "SPID"},
+        "title": {"sheet": "tlkpSpecialism", "column": "SPSPEC"},
+    },
+    "RecruitmentMethod": {
+        "id": {"sheet": "tlkpEntryType", "column": "ENID"},
+        "title": {"sheet": "tlkpEntryType", "column": "ENENTR"},
+    },
+    "Contact": {
+        "id": {"sheet": "tlkpContacts", "column": "CTID"},
+        "forename": {"sheet": "tlkpContacts", "column": "CTFNAME"},
+        "surname": {"sheet": "tlkpContacts", "column": "CTSNAME"},
+        "email": {"sheet": "tlkpContacts", "column": "CTEML"},
+        "phone": {"sheet": "tlkpContacts", "column": "CTPHONE"},
+        "onefile_username": {"sheet": "tlkpContacts", "column": "CTOFUN"},
+        "onefile_id": {"sheet": "tlkpContacts", "column": "CTOFUID"},
+    },
+    "ContactType": {
+        "id": {"sheet": "tlkpContactType", "column": "NTID"},
+        "title": {"sheet": "tlkpContactType", "column": "NTTYPE"},
+    },
+    "AssessmentType": {
+        "id": {"sheet": "tlkpOSFAType", "column": "OTID"},
+        "title": {"sheet": "tlkpOSFAType", "column": "OTTYPE"},
+    },
+    "Outcome": {
+        "id": {"sheet": "tlkpReviewOutcomes", "column": "AOID"},
+        "title": {"sheet": "tlkpReviewOutcomes", "column": "AOCODE"},
+    },
+    "ExitAssessmentOutcome": {
+        "id": {"sheet": "tlkpOSFAResult", "column": "RSID"},
+        "title": {"sheet": "tlkpOSFAResult", "column": "RSLT"},
+    },
+    "ExitAssessmentOutcome": {
+        "id": {"sheet": "tlkpOSFAType", "column": "OTID"},
+        "title": {"sheet": "tlkpOSFAType", "column": "OTTYPE"},
+    },
```

Comment:
looks like you forgot to rename the table name

Reply:
I have updated it in the most recent version